packages/angular_bloc/example/example.dart

```diff
@@ -1,69 +1,71 @@
+// This example is one-pager for the [angular_bloc](https://github.com/felangel/bloc/tree/master/examples/angular_counter) example.
+
 import 'dart:async';
 
-import 'package:angular/angular.dart';
+import 'package:angular/angular.dart';	
```

Comment:
can we revert all this additional whitespace that was added?

Reply:
Not sure what happened in there 😲 Whitespace party? 🎉 
Thanks!